### PR TITLE
Mevent server

### PR DESCRIPTION
**Pull request by PikalaxALT**
i'm guessing mevent_server_ish is the client end handler?

### CODE REVIEW

File: include/mevent_client.h [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#ifndef GUARD_MEVENT_SERVER_ISH_H
```
Comment (huderlem) [inline include/mevent_client.h]: Use updated header guard names

File: sym_ewram.txt [hunk 2]
```diff
@@ -12,14 +12,11 @@
     .include "src/union_room_battle.o"
 	.include "src/mevent2.o"
 	.include "src/mevent_801BAAC.o"
+    .include "src/mevent_server.o"
```
Comment (huderlem) [inline sym_ewram.txt]: The rest of the file uses tabs

File: src/scrcmd.c [hunk 3]
```diff
@@ -2217,7 +2217,7 @@ bool8 ScrCmd_checkmonobedience(struct ScriptContext *ctx)
 
 bool8 ScrCmd_cmdCF(struct ScriptContext *ctx)
```
Comment (huderlem) [inline src/scrcmd.c]: Can you document this function?  It appears `gotoram` is misnamed, and should be something like `returnram`.

Comment (PikalaxALT) [reply src/scrcmd.c]: Resolved in e30ea04

File: sym_ewram.txt (hunk 2 above)
Comment (PikalaxALT) [reply sym_ewram.txt]: Resolved in e30ea04

File: include/mevent_client.h (hunk 1 above)
Comment (PikalaxALT) [reply include/mevent_client.h]: Resolved in e30ea04

**Pull request merged**